KomaMRIIO/src/RawAcquisitionData/ISMRMRD.jl

```diff
@@ -222,8 +222,19 @@ Base.show(io::IO, raw::RawAcquisitionData) = begin
 end
 
 """
-Overwrites auxiliary function from KomaMRICore
+Define simulation output for MRD (or RawAcquisitionData)
 """
-function simulation_output(sig, seq, phantom_name, sys, sim_params)
-    return signal_to_raw_data(sig, seq; phantom_name, sys, sim_params)
+struct MRD <: SimulationOutput end
+
+function simulation_output(return_type::MRD; kwargs...)
+    sim_params_raw = copy(kwargs[:sim_params])
+    sim_params_raw["return_type"] = string(kwargs[:sim_params]["return_type"])
+    sim_params_raw["sim_method"] = string(kwargs[:sim_params]["sim_method"])
+    sim_params_raw["gpu"] = kwargs[:sim_params]["gpu"]
+    sim_params_raw["Nthreads"] = kwargs[:sim_params]["Nthreads"]
+    sim_params_raw["t_sim_parts"] = kwargs[:t_sim_parts]
+    sim_params_raw["type_sim_parts"] = kwargs[:excitation_bool]
+    sim_params_raw["Nblocks"] = kwargs[:Nparts]
+    sim_params_raw["sim_time_sec"] = kwargs[:timed_tuple_time]
+    out = signal_to_raw_data(kwargs[:sig], kwargs[:seq]; phantom_name=kwargs[:obj].name, sys=kwargs[:sys], sim_params=sim_params_raw)
```

Comment:
Change this.

Reply:
We rolled back the recent changes due to technical issues. Addressed in [dfda824](https://github.com/cncastillo/KomaMRI.jl/commit/dfda8249dd5d5d993ee0f6a27eb50f1fcdfdb4ee)